packages/ui/src/components/vuestic-components/va-select/VaSelectOptionList/VaSelectOptionList.vue

```diff
@@ -172,6 +172,12 @@ export default class VaSelectOptionList extends mixins(
     } else {
       this.hoveredOptionComputed = null
     }
+    // We need this to select list items when VaSelect has searchBar
+    // Presumably because searchBar gets focus and after clicking on OptionList line 10: @focus="hoverFirstOption" gets triggered
+    // Looks like this results in registering event listeners for only first visible items in the OptionList so items lower in the list cant be selected
+    this.focus()
```

Comment:
You don't need such a hack with refocusing. If you want to leave the focus on the options list, simply remove the focusing search bar after option selection.

Reply:
Thanks that's exactly what I wanted to discuss during the call, just a work in progress comment.